LDMP/gui/WidgetSelectArea.ui

```diff
@@ -522,6 +507,20 @@
         </layout>
        </widget>
       </item>
+      <item>
+       <layout class="QHBoxLayout" name="horizontalLayout_2">
+        <item>
+         <widget class="QLabel" name="label_2">
+          <property name="text">
+           <string>Settings name:</string>
```

Comment:
```suggestion
           <string>Area name:</string>
```

I'm confused about the label you used for the tag of the Area.
Wait for resolve this to merge

Reply:
ok let me change it.